DigitalLearningSolutions.Web/Views/MyAccount/_AccountSummaryCard.cshtml

```diff
@@ -28,6 +28,14 @@
           <partial name="_SummaryFieldValue" model="@Model.DelegateNumber" />
         </div>
       }
+      @if (Model.AliasId?.Length > 0) {

```

Comment:
Any particular reason we are using this instead of something like string.IsNullOrWhitespace()?

Reply:
Just because it matched the above one